monero-wallet-generator.html

```diff
@@ -10743,20 +10742,17 @@ <h4 style="color: #555;">Offline wallet generator</h4>
                 <input class="btn orange" type="button" onclick="js:genwallet(null);"
                        value="Generate wallet" action=""/>
                 <p>Custom entropy (leave empty to use PRNG)</p>
-                <input type="text" value="" id="user_entropy_widget" />
+                <input type="text" value="" id="user_entropy_widget" onKeyPress="if (event.keyCode == 13) js:genwallet(null);" />
         </div>
         <div class="col s1 offset-s1"><h4>or</h4></div>
         <div class="col s5 offset-s1">
-            <form>
```

Comment:
Why are these removed ?

Reply:
I'm concerned about this too as this html file has no doctype and therefore may not be interpreted as HTML5 https://stackoverflow.com/a/22328498/1117929

Form elements in older html specs need to have a <form> element parent.

Reply:
It was removed because if you let it, when you press enter key (instead of click on "generate wallet with prefix" button), the page is reloaded and the wallet is not created.
The page don't need to connect to a server so I don't think form is needed (https://stackoverflow.com/questions/8568660/is-there-any-danger-to-using-input-fields-outside-without-forms-in-html-javascri) but I could be wrong, I'm absolutely not a javascript expert. I only read the manual.
About the doctype, i'm too inexperienced to answer efficiently.

Reply:
That all looks OK to me. If there's a problem with a doctype, it can be added later.